lighthouse-docker/Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM node:9-slim
+
+LABEL maintainer="Eric Bidelman <ebidel@>"
+
+# Install utilities
+RUN apt-get update --fix-missing && apt-get -y upgrade
+
+# Install latest chrome dev package and fonts to support major charsets
+# (Chinese, Japanese, Arabic, Hebrew, Thai and a few others)
+# https://www.ubuntuupdates.org/package/google_chrome/stable/main/base/google-chrome-unstable
+RUN wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - \
+    && sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list' \
+    && apt-get update \
+    && apt-get install -y google-chrome-unstable fonts-ipafont-gothic fonts-wqy-zenhei fonts-thai-tlwg fonts-kacst ttf-freefont \
+      --no-install-recommends \
+    && rm -rf /var/lib/apt/lists/* \
+    && apt-get purge --auto-remove -y curl \
+    && rm -rf /src/*.deb
+
+ADD https://github.com/Yelp/dumb-init/releases/download/v1.2.1/dumb-init_1.2.1_amd64 /usr/local/bin/dumb-init
+RUN chmod +x /usr/local/bin/dumb-init
+
+# Download latest Lighthouse from npm.
+# Cache bust so we always get the latest version of LH when building the image.
+ARG CACHEBUST=1
+RUN yarn global add lighthouse
+
+# Install npm deps (express, etc.)
+COPY package.json yarn.lock /
+RUN yarn --frozen-lockfile --no-cache --production
+
+# Add the web server.
+COPY server.js /
+RUN chmod +x /server.js
+
+# Add entrypoint script.
+COPY entrypoint.sh /
+RUN chmod +x /entrypoint.sh
+
+# Add a chrome user and setup home dir.
+RUN groupadd --system chrome && \
+    useradd --system --create-home --gid chrome --groups audio,video chrome && \
+    mkdir --parents /home/chrome/reports && \
+    chown --recursive chrome:chrome /home/chrome
+
+USER chrome
+
+# Uncomment if you want to use volumes.
+#VOLUME /home/chrome/reports
+#WORKDIR /home/chrome/reports
+
+# Disable Lighthouse error reporting (prevents CLI prompt).
+ENV CI=true
```

Comment:
FWIW you could also do something more explicit than this like `--no-enable-error-reporting`

Reply:
would you be against switching to `--no-enable-error-reporting`? :)

Reply:
I went ENV var b/c the flag would need to added in a couple of different places:

https://github.com/GoogleChrome/lighthouse/pull/4864/files#diff-40aa323dd59e65755424b2a4a6b4bb7eR20
https://github.com/GoogleChrome/lighthouse/pull/4864/files#diff-314aa113831df7f13bc95d96f2a092edR48
https://github.com/GoogleChrome/lighthouse/pull/4864/files#diff-2141517e02e4b4f689469490f4a15ca8R33
...

When running the container, it's also easy for users to forget about it:

```
docker run -it --rm GoogleChrome/lighthouse https://example.com # Oops I get stuck waiting for the prompt.
```

Better to have it at the image-level IMO. Then, if someone really wants to flip it on, they can override the value in one place:

``` 
docker run -it --rm GoogleChrome/lighthouse -e CI=false https://example.com 
```

Is there a reason you like the flag more?

Reply:
Well the flag is just more explicit in why the env var is actually being used

Hypothetically `CI=true` may have other unintentional side effects we don't want since it implies "this thing is being tested" rather than "this thing is not interactive"